src/backend/metal/src/command.rs

```diff
@@ -842,7 +842,9 @@ impl CommandSink {
                     //Note: the original descriptor belongs to the framebuffer,
                     // and will me mutated afterwards.
                     desc: unsafe {
-                        msg_send![descriptor, copy]
+                        let desc: metal::RenderPassDescriptor = msg_send![descriptor, copy];
+                        msg_send![desc.as_ptr(), retain];
```

Comment:
Alternatively I suppose we could have `retained_descriptor` or similar (to be consistent)